BiometricKit/BiometricKit.h

```diff
@@ -1,13 +1,29 @@
 #import "BiometricKitDelegate.h"
+#import "BiometricKitIdentity.h"
 
+API_AVAILABLE(ios(7.0))
 @interface BiometricKit : NSObject
 
 + (BiometricKit *)manager;
 
 @property (nonatomic, assign) id<BiometricKitDelegate> delegate;
+@property (assign) BOOL inUse;
 
-- (NSArray *)identities:(id)object;
+// research on iOS 10.2 shows `options` supports { "BKFilterIdentity" : BiometricKitIdentity }
+- (NSArray<BiometricKitIdentity *> *)identities:(NSDictionary *)options;
 
-- (BOOL)isTouchIDCapable;
+- (float)getModulationRatio;
```

Comment:
Maybe `CGFloat`?

Reply:
This dump is from a 64-bit runtime; I do not believe this method should return a `CGFloat` (`typedef` to `double` on 64-bit environments)

Reply:
It often makes sense if a method is still `float`. You know, not everything uses `CGFloat` especially in low-level API.

Reply:
I asked because generally, class dumpers don't make this distinction and I don't know if these headers were hand made. I'm not sure if BiometricKit is a low level API.

Reply:
Understandable. It’s tricky sometimes unless `float` is really what you get from class-dumping in 64-bit.